addon/components/file-browser/component.js

```diff
@@ -472,7 +472,7 @@ export default Ember.Component.extend(Analytics, {
     },
     _createComponent(title) {
         return this.get('node')
-            .addChild(title, null, null, true)
+            .addChild(title, null, '', true)
```

Comment:
I don't understand, why do we need need/want `category: ''`? According to [API docs](https://developer.osf.io/#operation/nodes_create) that's invalid, but it sounds like it actually sets `category: 'uncategorized'`? If that's right, can we set that explicitly instead?

Reply:
Oh, never mind, [I see](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/settings/defaults.py#L255) ([twice?](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L213)).

Reply:
Nope, ['' == 'Uncategorized'](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L213)

Reply:
<img width="738" alt="screen shot 2018-02-02 at 4 38 01 pm" src="https://user-images.githubusercontent.com/348630/35755994-839076f6-0837-11e8-895d-e66fd0f8a143.png">